[2023-08-28 20:08:42,393] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-07-31T00:00:00+00:00 [queued]>
[2023-08-28 20:08:42,466] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-07-31T00:00:00+00:00 [queued]>
[2023-08-28 20:08:42,469] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:08:42,471] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-28 20:08:42,472] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:08:42,562] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-07-31 00:00:00+00:00
[2023-08-28 20:08:42,697] {standard_task_runner.py:52} INFO - Started process 24903 to run task
[2023-08-28 20:08:42,765] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta', 'scheduled__2023-07-31T00:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmp32_rolzc', '--error-file', '/tmp/tmpd4uljhwo']
[2023-08-28 20:08:42,779] {standard_task_runner.py:80} INFO - Job 12: Subtask cria_pasta
[2023-08-28 20:08:43,121] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-07-31T00:00:00+00:00 [running]> on host FelipeBreve-PC.localdomain
[2023-08-28 20:08:43,380] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-07-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-31T00:00:00+00:00
[2023-08-28 20:08:43,390] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-28 20:08:43,393] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/felipe/datapipeline/semana=2023-08-07"']
[2023-08-28 20:08:43,488] {subprocess.py:85} INFO - Output:
[2023-08-28 20:08:43,553] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-28 20:08:43,691] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta, execution_date=20230731T000000, start_date=20230828T230842, end_date=20230828T230843
[2023-08-28 20:08:43,767] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-28 20:08:43,929] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-28 20:17:55,759] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-07-31T00:00:00+00:00 [queued]>
[2023-08-28 20:17:55,786] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-07-31T00:00:00+00:00 [queued]>
[2023-08-28 20:17:55,790] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:17:55,791] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-28 20:17:55,791] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:17:55,848] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-07-31 00:00:00+00:00
[2023-08-28 20:17:55,931] {standard_task_runner.py:52} INFO - Started process 28626 to run task
[2023-08-28 20:17:55,969] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta', 'scheduled__2023-07-31T00:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmp51_ah15s', '--error-file', '/tmp/tmpo97_66x5']
[2023-08-28 20:17:55,980] {standard_task_runner.py:80} INFO - Job 24: Subtask cria_pasta
[2023-08-28 20:17:56,225] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-07-31T00:00:00+00:00 [running]> on host FelipeBreve-PC.localdomain
[2023-08-28 20:17:56,366] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-07-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-31T00:00:00+00:00
[2023-08-28 20:17:56,369] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-28 20:17:56,374] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/felipe/datapipeline/semana=2023-08-07"']
[2023-08-28 20:17:56,445] {subprocess.py:85} INFO - Output:
[2023-08-28 20:17:56,510] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-28 20:17:56,579] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta, execution_date=20230731T000000, start_date=20230828T231755, end_date=20230828T231756
[2023-08-28 20:17:56,630] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-28 20:17:56,725] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-28 20:21:37,574] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-07-31T00:00:00+00:00 [queued]>
[2023-08-28 20:21:37,604] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-07-31T00:00:00+00:00 [queued]>
[2023-08-28 20:21:37,605] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:21:37,605] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-28 20:21:37,605] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:21:37,643] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-07-31 00:00:00+00:00
[2023-08-28 20:21:37,692] {standard_task_runner.py:52} INFO - Started process 30428 to run task
[2023-08-28 20:21:37,713] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta', 'scheduled__2023-07-31T00:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmpj9_thqmt', '--error-file', '/tmp/tmpk1isuxlo']
[2023-08-28 20:21:37,718] {standard_task_runner.py:80} INFO - Job 24: Subtask cria_pasta
[2023-08-28 20:21:37,871] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-07-31T00:00:00+00:00 [running]> on host FelipeBreve-PC.localdomain
[2023-08-28 20:21:38,019] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-07-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-31T00:00:00+00:00
[2023-08-28 20:21:38,022] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-28 20:21:38,023] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/felipe/datapipeline/semana=2023-08-07"']
[2023-08-28 20:21:38,108] {subprocess.py:85} INFO - Output:
[2023-08-28 20:21:38,158] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-28 20:21:38,237] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta, execution_date=20230731T000000, start_date=20230828T232137, end_date=20230828T232138
[2023-08-28 20:21:38,297] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-28 20:21:38,384] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-28 20:25:22,641] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-07-31T00:00:00+00:00 [queued]>
[2023-08-28 20:25:22,677] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-07-31T00:00:00+00:00 [queued]>
[2023-08-28 20:25:22,678] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:25:22,678] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-28 20:25:22,678] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:25:22,738] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-07-31 00:00:00+00:00
[2023-08-28 20:25:22,815] {standard_task_runner.py:52} INFO - Started process 31968 to run task
[2023-08-28 20:25:22,855] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta', 'scheduled__2023-07-31T00:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmprhd225_5', '--error-file', '/tmp/tmp7quimgn2']
[2023-08-28 20:25:22,860] {standard_task_runner.py:80} INFO - Job 24: Subtask cria_pasta
[2023-08-28 20:25:23,092] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-07-31T00:00:00+00:00 [running]> on host FelipeBreve-PC.localdomain
[2023-08-28 20:25:23,266] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-07-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-31T00:00:00+00:00
[2023-08-28 20:25:23,272] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-28 20:25:23,273] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/felipe/datapipeline/semana=2023-08-07"']
[2023-08-28 20:25:23,354] {subprocess.py:85} INFO - Output:
[2023-08-28 20:25:23,400] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-28 20:25:23,475] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta, execution_date=20230731T000000, start_date=20230828T232522, end_date=20230828T232523
[2023-08-28 20:25:23,521] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-28 20:25:23,600] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-28 20:27:39,038] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-07-31T00:00:00+00:00 [queued]>
[2023-08-28 20:27:39,072] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-07-31T00:00:00+00:00 [queued]>
[2023-08-28 20:27:39,073] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:27:39,073] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-28 20:27:39,073] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:27:39,125] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-07-31 00:00:00+00:00
[2023-08-28 20:27:39,183] {standard_task_runner.py:52} INFO - Started process 206 to run task
[2023-08-28 20:27:39,202] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta', 'scheduled__2023-07-31T00:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmputhatlr4', '--error-file', '/tmp/tmpqvi0vyxa']
[2023-08-28 20:27:39,208] {standard_task_runner.py:80} INFO - Job 24: Subtask cria_pasta
[2023-08-28 20:27:39,356] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-07-31T00:00:00+00:00 [running]> on host FelipeBreve-PC.localdomain
[2023-08-28 20:27:39,519] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-07-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-31T00:00:00+00:00
[2023-08-28 20:27:39,524] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-28 20:27:39,527] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/felipe/datapipeline/semana=2023-08-07"']
[2023-08-28 20:27:39,625] {subprocess.py:85} INFO - Output:
[2023-08-28 20:27:39,760] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-28 20:27:39,927] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta, execution_date=20230731T000000, start_date=20230828T232739, end_date=20230828T232739
[2023-08-28 20:27:40,034] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-28 20:27:40,333] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-28 20:29:57,609] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-07-31T00:00:00+00:00 [queued]>
[2023-08-28 20:29:57,638] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-07-31T00:00:00+00:00 [queued]>
[2023-08-28 20:29:57,638] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:29:57,638] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-28 20:29:57,638] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:29:57,681] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-07-31 00:00:00+00:00
[2023-08-28 20:29:57,734] {standard_task_runner.py:52} INFO - Started process 1723 to run task
[2023-08-28 20:29:57,756] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta', 'scheduled__2023-07-31T00:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmp5qlzi8d8', '--error-file', '/tmp/tmpv3xfe6a6']
[2023-08-28 20:29:57,765] {standard_task_runner.py:80} INFO - Job 24: Subtask cria_pasta
[2023-08-28 20:29:57,942] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-07-31T00:00:00+00:00 [running]> on host FelipeBreve-PC.localdomain
[2023-08-28 20:29:58,081] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-07-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-31T00:00:00+00:00
[2023-08-28 20:29:58,085] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-28 20:29:58,086] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/felipe/datapipeline/semana=2023-08-07"']
[2023-08-28 20:29:58,201] {subprocess.py:85} INFO - Output:
[2023-08-28 20:29:58,266] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-28 20:29:58,426] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta, execution_date=20230731T000000, start_date=20230828T232957, end_date=20230828T232958
[2023-08-28 20:29:58,533] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-28 20:29:58,687] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-28 20:40:11,755] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-07-31T00:00:00+00:00 [queued]>
[2023-08-28 20:40:11,786] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-07-31T00:00:00+00:00 [queued]>
[2023-08-28 20:40:11,787] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:40:11,787] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-28 20:40:11,787] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:40:11,841] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-07-31 00:00:00+00:00
[2023-08-28 20:40:11,921] {standard_task_runner.py:52} INFO - Started process 4489 to run task
[2023-08-28 20:40:11,958] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta', 'scheduled__2023-07-31T00:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmpfjlyhwgs', '--error-file', '/tmp/tmpaoigd0zc']
[2023-08-28 20:40:11,970] {standard_task_runner.py:80} INFO - Job 24: Subtask cria_pasta
[2023-08-28 20:40:12,245] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-07-31T00:00:00+00:00 [running]> on host FelipeBreve-PC.localdomain
[2023-08-28 20:40:12,551] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-07-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-31T00:00:00+00:00
[2023-08-28 20:40:12,558] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-28 20:40:12,560] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/felipe/datapipeline/semana=2023-08-07"']
[2023-08-28 20:40:12,709] {subprocess.py:85} INFO - Output:
[2023-08-28 20:40:12,793] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-28 20:40:12,896] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta, execution_date=20230731T000000, start_date=20230828T234011, end_date=20230828T234012
[2023-08-28 20:40:12,985] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-28 20:40:13,078] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
